lms/djangoapps/instructor/views/api.py

```diff
@@ -1351,12 +1352,6 @@ def get_students_features(request, course_id, csv=False):  # pylint: disable=red
         query_features.append('team')
         query_features_names['team'] = _('Team')
 
-    # For compatibility reasons, city and country should always appear last.
-    query_features.append('city')
-    query_features_names['city'] = _('City')
-    query_features.append('country')
-    query_features_names['country'] = _('Country')
```

Comment:
@kaizoku I don't know if this really matters, but I think it would be better to leave the Country field at the end, just to keep the changes here as minimal as possible. In other words, instead of reverting that past commit, just remove the city field.

Reply:
@bradenmacdonald  Yep, that makes sense from a total change perspective and to maintain backwards compatibility as much as possible.
I added another commit which re-introduces the country field at the end instead. Should I merge these two to remove the revert so we just have one commit here?

Reply:
Perfect, thanks! Yes, please squash to one commit and rebase, then I can approve and get this merged.

Reply:
Thanks @bradenmacdonald, I've squashed, written a better commit message, and rebased against master. This should be ready for merge now.